overview/templates/overview/index.html

```diff
@@ -31,20 +31,50 @@
 
 {% block content %} 
     <!-- Main component for a primary marketing message or call to action -->
-    <div class="jumbotron row">
-        <div class="col-md-6">
-            <h1>Vote Local!</h1>
-            <p>If you want more information before you cast your 2017 ballot for Cambridge City Council, you've come to the right place. We're compiling everything we can find - from op-eds to campaign finance records. Determine who deserves your #1 vote - or your #{{ num_runners }}!</p>
-            <p>
-                <a class="btn btn-lg btn-primary" href="{% url 'all' %}" role="button">Explore Candidates &raquo;</a>
-            </p>
-        </div>
-        <div class="col-md-6">
-            <div id="map" class="map-container">
-                <div class="map-alert">
-                    <div class="alert alert-warning" data-bind="text:gerror"></div>
+    <div class="jumbotron" style="margin-top: -20px">
+        <div class="container">
+            <div class="col-md-6">
+                <h1 style="text-align:center">Vote Local!</h1>
+                <p>We're compiling everything we can find about 2017 Cambridge City Council candidates - from op-eds to campaign finance records. Determine who deserves your #1, #2, or #9 vote - you've got {{ num_runners }} options!</p>
+                <p style="text-align:center">
+                    <a class="btn btn-lg btn-primary" href="{% url 'all' %}" role="button">Explore Candidates &raquo;</a>
+                </p>
+            </div>
+            <div class="col-md-6">
+                <div id="map" class="map-container">
+                    <div class="map-alert">
+                        <div class="alert alert-warning" data-bind="text:gerror"></div>
+                    </div>
                 </div>
             </div>
         </div>
     </div>
+
+
+    <div class="container">
+        <div class="row">
+            <div class="col-md-4">
+                <h2>Voting</h2>
+                <p>Cambridge has a ranked choice system, which might be different than what you're used to. We've made an illustration of how it works as well as collected other explanations.</p>
+                <p>
+                    <a class="btn btn-default" href="{% url 'how_to_vote' %}" role="button">Learn more &raquo;</a>
+                    <a class="btn btn-default" href="https://www.sec.state.ma.us/ovr/" role="button">Register to vote &raquo;</a>
```

Comment:
maybe `target="_blank" rel="noopener"`

Reply:
I think role="link" or no role probably makes more sense than button which I think is normally used for elements that have javascript actions that trigger something like "submit"